<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>527</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="patientSFNameLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Patient's FName</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="patientSFNameLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="patientSLNameLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Patient's LName</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="patientSLNameLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonSearch">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search Patients</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>841</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>P's FName</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P's LName</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P's PNumber</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ex's Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ex's Fee</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ex's Diagnosis</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ex's 2ndDate</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tr's StartDate</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tr's EndDate</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tr's Result</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
